Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -741,6 +740,8 @@ def fetch_incidents(client: Client,
 
 
 def main():
+    dbfile = NamedTemporaryFile(delete=False)
```

Comment:
I would use a fixed file so it is the same between requests. Otherwise we are loosing the caching (not sure how important it is). Use tempfile.gettempdir() and a fixed file name.